### DISCUSSION TITLE
possibly dumb question.. translating placeholder in input tag?

### DISCUSSION THREAD

**Discussion by baradhili** [intlify/vue-i18n#430] [Q&A] [answered]
Scratching my head over how to make this translated
```
<input class="form-control" 
    type="password" 
    placeholder="Password"
    id="txtPassword"
    v-model="password"
>
```
I'd like to translate the placeholder.. but of course if I do `{{ $t("login.password") }}` then I'll get server errors around attribute value errors..

**Comment by danielpetrica** [comment, marked as answer]
you can use `:placeholder="$t('login.password')"`, using `:` in front of attributes names let's you use functions and variables inside the **this** and **global** scope

**Comment by baradhili** [comment]
:) thanks was thinking in a completely different direction ....
